autogen/oai/client.py

```diff
@@ -275,21 +277,32 @@ def create(self, params: Dict[str, Any]) -> ChatCompletion:
 
         return response
 
-    def cost(self, response: Union[ChatCompletion, Completion]) -> float:
+    def cost(
+        self,
+        response: Union[ChatCompletion, Completion],
+        token_cost_1k: Optional[Union[float, Dict[str, float]]] = None,
+    ) -> float:
```

Comment:
Not critical, yet it might be reasonable to switch to Decimal type (it smells to have float for currency/money values :) - openai_utils.py included